src/olympia/users/models.py

```diff
@@ -1146,3 +1146,15 @@ def watch_changes(old_attr=None, new_attr=None, instance=None, sender=None, **kw
 
 
 user_logged_in.connect(UserProfile.user_logged_in)
+
+
+class FxaToken(ModelBase):
```

Comment:
I wonder how we are going to clean that table once it fills up...

Reply:
we could clear duplicates for user if they've not been updated in x amount of time.  I don't have a good idea for it really.

Reply:
logged #18280 - if we're refreshing the token we'll have to compare `modified` rather then `created`